presto-verifier/src/test/java/com/facebook/presto/verifier/framework/TestVerificationManager.java

```diff
@@ -224,4 +227,251 @@ private VerificationManager getVerificationManager(List<SourceQuery> sourceQueri
                 new QueryConfigurationOverridesConfig(),
                 verifierConfig);
     }
+
+    public static class TestLimitWithoutOrderByChecker
+    {
+        @Test
+        public void testQueryWithTableSampleToPass()
+        {
+            final String query = "INSERT INTO output_table (\"col1\", \"col2\", \"col3\", \"imps\", \"ds\") " +
+                    "SELECT " +
+                    "  col1, col2, col3, imps, '2021-11-20' AS ds " +
+                    "FROM (" +
+                    "  SELECT col1, col2, col3, COUNT(1) AS imps " +
+                    "  FROM " +
+                    "    source_table TABLESAMPLE BERNOULLI(0.1) " +
+                    "  WHERE " +
+                    "    ds = '2021-11-20' " +
+                    "    AND pair_id IS NOT NULL " +
+                    "  GROUP BY 1, 2, 3 " +
+                    ")";
+            final Statement statement = SQL_PARSER.createStatement(query, PARSING_OPTIONS);
+            final VerificationManager.LimitWithoutOrderByChecker checker = new VerificationManager.LimitWithoutOrderByChecker();
+            final Boolean result = checker.process(statement, 0);
```

Comment:
remove all the `final` keywords in the tests. We don't usually use final in the body of methods